tests/e2e/query_modules/query_modules.py

```diff
@@ -0,0 +1,95 @@
+# License, and you may not use this file except in compliance with the Business Source License.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0, included in the file
+# licenses/APL.txt.
+
+import sys
+
+import pytest
+from common import connect, execute_and_fetch_all
+
+
+def test_convert_list1():
+    cursor = connect().cursor()
+    result = execute_and_fetch_all(cursor, f"RETURN convert.str2object('[2, 4, 8, [2]]') AS result;")[0][0]
+    assert (result) == [2, 4, 8, [2]]
+
+
+def test_convert_list_wrong():
+    cursor = connect().cursor()
+    result = execute_and_fetch_all(cursor, f"RETURN convert.str2object('[2, 4, 8, [2]]') AS result;")[0][0]
+    assert (result) != [3, 4, 8, [2]]
+
+
+def test_mgps1():
+    cursor = connect().cursor()
+    result = list(
+        execute_and_fetch_all(
+            cursor, f"CALL mgps.components() YIELD edition, name, versions RETURN edition, name, versions;"
+        )[0]
+    )
+    assert (result) == ["community", "Memgraph", ["5.9.0"]]
+
+
+def test_node_type_properties1():
+    cursor = connect().cursor()
+    execute_and_fetch_all(
+        cursor,
+        "CREATE (d:Dog {name: 'Rex', owner: 'Carl'})-[l:LOVES]->(a:Activity {name: 'Running', location: 'Zadar'})",
+    )
+    result = list(
+        execute_and_fetch_all(
+            cursor,
+            f"CALL libschema.node_type_properties() YIELD nodeType, nodeLabels, propertyName, propertyTypes , mandatory RETURN nodeType, nodeLabels, propertyName, propertyTypes , mandatory ORDER BY propertyName, nodeLabels[0];",
+        )[0]
+    )
+    assert (result) == [":`Activity`", ["Activity"], "location", ["String"], True]
+
+    result = list(
+        execute_and_fetch_all(
+            cursor,
+            f"CALL libschema.node_type_properties() YIELD nodeType, nodeLabels, propertyName, propertyTypes , mandatory RETURN nodeType, nodeLabels, propertyName, propertyTypes , mandatory ORDER BY propertyName, nodeLabels[0];",
+        )[1]
+    )
+    assert (result) == [":`Activity`", ["Activity"], "name", ["String"], True]
+
+    result = list(
+        execute_and_fetch_all(
+            cursor,
+            f"CALL libschema.node_type_properties() YIELD nodeType, nodeLabels, propertyName, propertyTypes , mandatory RETURN nodeType, nodeLabels, propertyName, propertyTypes , mandatory ORDER BY propertyName, nodeLabels[0];",
+        )[2]
+    )
+    assert (result) == [":`Dog`", ["Dog"], "name", ["String"], True]
+
+    result = list(
+        execute_and_fetch_all(
+            cursor,
+            f"CALL libschema.node_type_properties() YIELD nodeType, nodeLabels, propertyName, propertyTypes , mandatory RETURN nodeType, nodeLabels, propertyName, propertyTypes , mandatory ORDER BY propertyName, nodeLabels[0];",
+        )[3]
+    )
+    assert (result) == [":`Dog`", ["Dog"], "owner", ["String"], True]
```

Comment:
Does it make sense to move this part to new file as this is just test for `node_type_properties`